packages/central-server/src/database/meditrakSyncQueue/MeditrakSyncRecordUpdater.js

```diff
@@ -15,7 +15,10 @@ export class MeditrakSyncRecordUpdater {
    * @public
    */
   async updateSyncRecords(changes) {
-    return Promise.all(changes.map(change => this.processChange(change)));
+    for (let i = 0; i < changes.length; i++) {
+      const change = changes[i];
+      await this.processChange(change);
+    }
```

Comment:
Just apply the changes one by one. In theory the deletes should always be in the change queue after the creates. 

Arguably a more thorough change could be made here (ie. check records exists when enqueueing an update), but historically it's been pretty stable so I feel this should suffice? Happy to be told to be less lazy by the reviewer 😉